pkg/chains/constants.go

```diff
@@ -14,24 +14,32 @@ limitations under the License.
 package chains
 
 const (
-	SignedMessagesCount     = "sgcount"
-	SignsStoredCount        = "stcount"
-	PayloadUploadeCount     = "plcount"
-	MarkedAsSignedCount     = "mrcount"
-	PipelineRunSignedName   = "pipelinerun_sign_created_total"
-	PipelineRunSignedDesc   = "Total number of signed messages for pipelineruns"
-	PipelineRunUploadedName = "pipelinerun_payload_uploaded_total"
-	PipelineRunUploadedDesc = "Total number of uploaded payloads for pipelineruns"
-	PipelineRunStoredName   = "pipelinerun_payload_stored_total"
-	PipelineRunStoredDesc   = "Total number of stored payloads for pipelineruns"
-	PipelineRunMarkedName   = "pipelinerun_marked_signed_total"
-	PipelineRunMarkedDesc   = "Total number of objects marked as signed for pipelineruns"
-	TaskRunSignedName       = "taskrun_sign_created_total"
-	TaskRunSignedDesc       = "Total number of signed messages for taskruns"
-	TaskRunUploadedName     = "taskrun_payload_uploaded_total"
-	TaskRunUploadedDesc     = "Total number of uploaded payloads for taskruns"
-	TaskRunStoredName       = "taskrun_payload_stored_total"
-	TaskRunStoredDesc       = "Total number of stored payloads for taskruns"
-	TaskRunMarkedName       = "taskrun_marked_signed_total"
-	TaskRunMarkedDesc       = "Total number of objects marked as signed for taskruns"
+	SignedMessagesCount          = "sgcount"
+	SignsStoredCount             = "stcount"
+	PayloadUploadeCount          = "plcount"
+	MarkedAsSignedCount          = "mrcount"
+	PipelineRunSignedName        = "pipelinerun_sign_created_total"
+	PipelineRunSignedDesc        = "Total number of signed messages for pipelineruns"
+	PipelineRunUploadedName      = "pipelinerun_payload_uploaded_total"
+	PipelineRunUploadedDesc      = "Total number of uploaded payloads for pipelineruns"
+	PipelineRunStoredName        = "pipelinerun_payload_stored_total"
+	PipelineRunStoredDesc        = "Total number of stored payloads for pipelineruns"
+	PipelineRunMarkedName        = "pipelinerun_marked_signed_total"
+	PipelineRunMarkedDesc        = "Total number of objects marked as signed for pipelineruns"
+	PipelineRunSignedMsg         = "pipelinerun_signed_messages"
```

Comment:
How's this different than `PipelineRunSignedName` ?

Reply:
The ones currently used produce totals across the entire cluster. We need a more detailed information, such as a namespace, taskname, success/failure and other kinds of labels attached to the counters. So my proposal is to leave the old ones alone and create a new time series that can be utilized to display info related to individual projects in Grafana or some other tool.

Reply:
That makes sense. Can we make the name and description indicative of this?

Reply:
Do we have a naming convention?

Reply:
Not an official one. I would just like to make it obvious that certain metric is namespace based. Does that make sense?

Reply:
Absolutely

Reply:
Does it look ok?

Reply:
Hi @lcarva do the names look ok to you?

Reply:
Yes, just wanted to make small and manageable changes at a time.